openmmapi/include/TorchForce.h

```diff
@@ -140,6 +144,20 @@ class OPENMM_EXPORT_NN TorchForce : public OpenMM::Force {
      * @param defaultValue   the default value of the parameter
      */
     void setGlobalParameterDefaultValue(int index, double defaultValue);
+    /**
+     * Set a value of a property.
+     *
+     * @param name           the name of the property
+     * @param value          the value of the property
+     */
+    void setProperty(const std::string& name, const std::string& value);
+    /**
+     * Get a value of a property.
+     *
+     * @param name           the name of the property
+     * @return the value of the property
+     */
+    const std::string& getProperty(const std::string& name) const;
```

Comment:
There's no way to get a list of what properties are defined.  You just have to somehow know their names to query them.  I suggest changing this to

```c++
const std::map<std::string, std::string>& getProperties() const;
```